plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtStoragePoolDef.java

```diff
@@ -124,10 +138,17 @@ public AuthenticationType getAuthType() {
         return _authType;
     }
 
+    public Map<String, String> getNfsOpts() {
+        return _nfsopts;
+    }
+
     @Override
     public String toString() {
         StringBuilder storagePoolBuilder = new StringBuilder();
-        if (_poolType == PoolType.GLUSTERFS) {
+        if (_poolType == PoolType.NETFS && _nfsopts != null) {
```

Comment:
switch statement?

Reply:
Switch could be used when only _poolType is being checked. I have tried to refactor the method using switch. Please check.